### DISCUSSION TITLE
Did version 9.5.1 bump up to version 9.6?

### DISCUSSION THREAD

**Discussion by thabaum** [dnnsoftware/Dnn.Platform#3678] [General] [answered]
I noticed the milestones recently adjusted from 9.5.1 to 9.6.  Does this mean 9.5.1 is being packaged up for release or did things get pushed forward into next version as I see it is already 100%?

**Comment by sleupold** [comment]
there had been features added to the develop branch, we decided to include within next release. Therefore it will be called 9.6.0 instead of 9.5.1 (preceeded by RC, of course.

**Reply by thabaum** [reply]
Should we expect another RC?

**Reply by bdukes** [reply]
DNN 9.6.0 RC1 should be released soon

**Comment by mitchelsellers** [comment, marked as answer]
9.5.1 is being skipped and 9.6.0 will be the next release.  RC will be out today!

**Reply by thabaum** [reply]
Great, thanks for the update!

**Comment by valadas** [comment]
Yes, RC1 is here https://github.com/dnnsoftware/Dnn.Platform/releases/tag/v9.6.0-rc1

It has a small upgrade issue, which does not make the site unusable, test away :)
We will publish RC2 very soon with a fix for those upgrades. If testing of RC2 goes fine, we should have it officially released 2 weeks from RC1, so in about a week and a half.

**Reply by thabaum** [reply]
Busy through the weekend, things might clear up next week I will test this release, thank you!
